docs/configure/triggers.md

```diff
@@ -39,6 +40,23 @@ To configure a trigger:
 Select the data types for which the Trigger should send requests.
```

Comment:
on line 35, change `   For the respective type, configure the respective attributes:` to `   For some types you can configure additional attributes:`. Otherwise people may wonder why their type isn't listed